main/qpdf/APKBUILD

```diff
@@ -1,19 +1,29 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=qpdf
-pkgver=7.0.0
+pkgver=7.1.1
 pkgrel=0
 pkgdesc="Command-line tools and library for transforming PDF files"
-url="http://qpdf.sourceforge.net/"
+url="http://qpdf.sourceforge.net"
 arch="all"
-license="Artistic-2.0"
+license="Apache-2.0"
 depends=""
 # coreutils needed for tests
 makedepends="zlib-dev libjpeg-turbo-dev bash coreutils perl"
```

Comment:
Can you move the tools needed for tests(check) to checkdepends?

Reply:
Yes sir.
Moved coreutils to checkdepends.